src/linagora.esn.calendar/app/planning/calendar-planning.controller.js

```diff
@@ -90,7 +93,7 @@ const _ = require('lodash');
       });
     }
 
-    function updateCalendar(event, calendar) {
+    function updateCalendar(calendar) {
```

Comment:
this change is wrong.
it is true that the `event` is not used here but it is needed to be the first argument and the calendar as the second argument.

if we remove it the function will use the event `CAL_EVENTS.CALENDARS.UPDATE` as a calendar here:
```js
if (calendar.getUniqueId() === cal.getUniqueId()) {
```

as if doing
```js
if (CAL_EVENTS.CALENDARS.UPDATE.getUniqueId() === cal.getUniqueId()) {
```

Reply:
ok , we must first check this part of code , this function i think it is never called